MantidQt/CustomInterfaces/src/Muon/MuonAnalysisResultTableTab.cpp

```diff
@@ -497,40 +497,47 @@ void MuonAnalysisResultTableTab::populateLogsAndValues(
     // Get log information
     std::string wsName = fittedWsList[i].toStdString();
     auto ws = retrieveWSChecked<ExperimentInfo>(wsName + WORKSPACE_POSTFIX);
-
-    Mantid::Kernel::DateAndTime start = ws->run().startTime();
-    Mantid::Kernel::DateAndTime end = ws->run().endTime();
-
     const std::vector<Property *> &logData = ws->run().getLogData();
-
+    const TimeSeriesProperty<bool> *runningLog; // defined here to keep the
+                                                // object in scope. If
+                                                // applicable it is assigned a
+                                                // value within an if statement.
+
+    std::string running = "running"; // define the running log via a string to
```

Comment:
I would suggest making this a static string constant at the top of the file to avoid repeated evaluations: Placing

```
const std::string RUNNING_LOG_NAME = "running"
```

in an anonymous namespace at the top of the file will achieve this.